jenkins-slave

```diff
@@ -45,6 +46,15 @@ else
 		fi ;;
 	esac
 
+	# if -workDir is not provided try env vars
+	if [ ! -z "$JENKINS_AGENT_WORKDIR" ]; then
+		case "$@" in
+			*"-workDir"*) echo "Warning: Work directory is defined twice in command-line arguments and the environment variable" ;;
```

Comment:
this construct is generally unreliable, but we cannot do it better without parsing CMD line